xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -465,7 +465,12 @@ CPVRRecordingPtr CPVRRecordings::GetRecordingForEpgTag(const EPG::CEpgInfoTagPtr
     else
     {
       // uid is optional, so check other relevant data.
-      if (recording.second->Channel() == epgTag->ChannelTag() &&
```

Comment:
Can you explain where the deadlocks comes from? which other thread takes a dependend lock?